### ISSUE TITLE
Contenu du site normes.transport.data.gouv.fr

### ISSUE THREAD

**Issue by TuThoThai** [etalab/transport-profil-netex-fr#74]
A la suite d'une session de travail au sein du GT7, voici une question ouverte pour les équipes du PAN.

Contexte :
- Elaboration d'un processus de publication des profils des normes européennes NeTEx et SIRI
- Documentation dudit processus

Rôles :
- Equipe du PAN : soutien technique et gestion technique des sites [normes ](https://normes.transport.data.gouv.fr/)et [schema](https://schema.data.gouv.fr/) en lien avec les données transport
- GT7 : processus de publication sur les sites gérés par les équipes du PAN, qui doit être étendu aux travaux de la CN 03

Questions du GT7 aux équipes du PAN :
- Le site normes.transport.data.gouv.fr ne devrait-il pas être uniquement consacré aux profils des normes européennes et internationales, dont NeTEx et SIRI ?
- Auquel cas, les trois derniers éléments de [ce ticket](https://github.com/etalab/transport-profil-netex-fr/issues/41) ne devraient-il pas être publiés sur un autre site (par exemple schema.data.gouv.fr) ?

**Comment by thbar** [comment]
Voir:
- https://github.com/datagouv/schema.data.gouv.fr/issues/308

**Comment by thbar** [comment]
Voir aussi:
- https://github.com/etalab/transport-profil-netex-fr/issues/78
- https://github.com/etalab/transport-profil-netex-fr/issues/79

**Comment by thbar** [comment]
Poke @gonzalezsuitt, issue intéressante à suivre !

**Comment by Brewennn** [comment]
Pour répondre aux questions : 

- Le site normes.transport.data.gouv.fr doit se consacrer avant tout aux profils France des normes puisque les articles 4 et 5 du règlement MMTIS oblige la publication des données a minima selon les profils nationaux lorsqu'une norme est utilisée. Le décret du 4 mars oblige l’utilisation des profils publiés sur normes.transport.data.gouv.fr (art. D. 1115-1 du code des transports) et donc il ne devrait pas y avoir les profils européens puisque leur utilisation est facultative. On pourrait ajouter les profils européens mais il faudrait changer la formulation de l'article en question.

- Les 3 derniers éléments cités (Chapeau MAS, standard covoiturage, standard Compte mobilité) ne doivent pas âtre affichés sur normes.transport.data.gouv.fr car les standards sortent du cadre de MMTIS. Par contre il serait intéressant de pouvoir indiqué quelque part (documentation PAN dans une section "pour aller plus loin" ?) l'existence de tous les standards.

Une mise à jour de la documentation du PAN est prévu et reprendra le premier point.

**Comment by TuThoThai** [comment]
Un grand merci @Brewennn pour la clarification. En ce cas, côté GT7 on retravaille le document de processus de rédaction, modification et publication des profils pour revenir intégralement dans le cadre MMTIS.

**Comment by Brewennn** [comment]
@TuThoThai et @thbar, je suis allé un peu vite en besogne sur le second point, je viens d'avoir connaissance d'un projet d'intégrer les standards quelque part dans les onglets de transport.data.gouv.fr et éventuellement dans normes dans un furtur projet d'arrêté.

**Comment by thbar** [comment]
@Brewennn yes, j'allais compléter, il y avait effectivement une demande du ministère d'intégrer les 3 autres éléments, voir:
- https://github.com/etalab/transport-profil-netex-fr/issues/35

Ce qui introduit d'ailleurs des choses techniquement à traiter lors de la refonte qu'on commence à étudier.

**Comment by TuThoThai** [comment]
@Brewennn et @thbar, c'est bien noté. Pour le coup, j'ai donc modifié légèrement le contenu du processus pour laisser la porte ouverte aux autres publications. 
Le processus est désormais dispo en MD et près à être mergé ici une fois officiellement validé en GT7 et CN 03. Vous le trouverez [ici ](https://github.com/TuThoThai/transport-profil-netex-fr/blob/Process-Publication/Process-Redaction-Modification-Publication.md)en attendant. :)

**Comment by thbar** [comment]
Merci @TuThoThai! Un point un peu structurant est le suivant: ce doc est actuellement sur le profil NeTEx, mais s'applique aussi à la gouvernance SIRI. Il faudra voir au final où on souhaite le stocker (pas forcément dans http://github.com/etalab/transport-normes-site, mais peut-être que oui...).

/cc @ptitfred on en reparle !

**Comment by thbar** [comment]
Et attention, le merge ici va se traduire par une publication immédiate, il faudra la tester avant !

**Comment by TuThoThai** [comment]
@thbar, tout à fait d'accord qu'il reste à définir où publier ce doc. J'ai profité d'avoir un fork de ce repo et d'une branche dédiée au process pour m'éviter plusieurs étapes d'ouverture de repo. :)

**Comment by TuThoThai** [comment]
> @Brewennn et @thbar, c'est bien noté. Pour le coup, j'ai donc modifié légèrement le contenu du processus pour laisser la porte ouverte aux autres publications. Le processus est désormais dispo en MD et près à être mergé ici une fois officiellement validé en GT7 et CN 03. Vous le trouverez [ici ](https://github.com/TuThoThai/transport-profil-netex-fr/blob/Process-Publication/Process-Redaction-Modification-Publication.md)en attendant. :)

Note : Le document a été modifié ce jour à la suite des retours de la CN03.
